modules/core/src/smithy4s/Endpoint.scala

```diff
@@ -53,6 +53,19 @@ trait Endpoint[Op[_, _, _, _, _], I, E, O, SI, SO] { outer =>
 
   def errorable: Option[Errorable[E]] = None
 
+  private[smithy4s] def mapHints(
+      f: Hints => Hints
+  ): Endpoint[Op, I, E, O, SI, SO] =
+    new Endpoint[Op, I, E, O, SI, SO] {
```

Comment:
This is missing the `errorable`

Reply:
the price of having `errorable` have a default implementation of `None` :( 

can we get rid of that?

Reply:
yeah , I think i used the IDE implement methods feature :)